Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,65 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings The selected Pokemon.
          */
         openPokemonMenuContext(settings: any): void {
+            var moves: BattleMovr.IMove[] = settings.pokemon.moves,
+                options: any[],
+                i: number;
+
+            for (i = 0; i < moves.length; i += 1) {
+                if (moves[i].title = "Cut") {
```

Comment:
Things like `console.log.bind(console)` are good for testing, but not for going upstream. This'll have to be changed into using the move's listing's callback.